src/TestProblem/Hydro/Plummer/ExtAcc_Plummer.cpp

```diff
@@ -8,36 +8,81 @@
 
 
 
+// =================================
+// I. Set an auxiliary array
+// =================================
+
+#ifndef __CUDACC__
+
+extern double Plummer_Rho0;
+extern double Plummer_R0;
+extern double Plummer_Center[3];
+extern double Plummer_ExtAccMFrac;
+
+//-------------------------------------------------------------------------------------------------------
+// Function    :  SetExtAccAuxArray_Plummer
+// Description :  Set the auxiliary array ExtAcc_AuxArray[] used by ExtAcc_Plummer()
+//
+// Note        :  1. Invoked by Init_ExtAcc_Plummer()
+//                2. AuxArray[] has the size of EXT_ACC_NAUX_MAX defined in Macro.h (default = 10)
```

Comment:
default = 20 for `EXT_POT_NAUX_MAX`, `EXT_ACC_NAUX_MAX`, and `EOS_NAUX_MAX`. same typos in other files.

Reply:
1. Fixed.
2. It's always defined in `CUPOT.h`. `Init_TestProb_ELBDM_ExtPot.cpp` doesn't really use `EXT_POT_NAUX_MAX`. Does compilation fail in some of your tests?

Reply:
2. Fixed